lib/db/mysql/schema.sql

```diff
@@ -43,3 +43,14 @@ CREATE TABLE IF NOT EXISTS tokens (
   scope VARCHAR(256) NOT NULL,
   createdAt TIMESTAMP NOT NULL DEFAULT CURRENT_TIMESTAMP
 ) ENGINE=InnoDB CHARACTER SET utf8 COLLATE utf8_unicode_ci;
+
+CREATE TABLE IF NOT EXISTS developers (
+  email VARCHAR(256) NOT NULL PRIMARY KEY,
```

Comment:
- clientId: 8
- userId: 16
- clientSecret: 32
- code: 32
- token: 32

We could probably get by with `BINARY(8)`, but if we wanted to be extra safe, we could use 16, like userId.